#error handling
'''
interrupting the normal execution of a code is error or exception
 to handle that errors we use error handling
'''
#error handlings
'''
try: #used to solve error incase of future
 risky code
except:#incase of error present case
    print('error')
else:#incase of no error in try block
    print('no error')
finally:#it is always prints in anycase
    print('always')
'''
#a=1
#print(b)
try:
    print(b)
except:
    print('error')
else:
    print('no error')
finally:
    print('always')
